doc/ref_model/chapters/chapter02.rst

```diff
@@ -260,112 +290,220 @@ Telco Edge is commonly coupled with 5G use cases, seen as one of the ingredients
-Very Low Latency - Edge Sites                    very-low-latency-edge   ✅                           ✅                                      Labels a host/node as located in an edge site, for workloads requiring low latency (specify value) to final users or geographical distribution.
-Ultra Low Latency - Edge Sites                   ultra-low-latency-edge  ✅                           ✅                                      Labels a host/node as located in an edge site, for workloads requiring low latency (specify value) to final users or geographical distribution.
-Fixed function accelerator                       compute-ffa             ❌                           ✅                                      Labels a host/node that includes a consumable fixed function accelerator (non-programmable, e.g. Crypto, vRAN-specific adapter).
-Firmware-programmable adapter                    compute-fpga            ❌                           ✅                                      Labels a host/node that includes a consumable Firmware-programmable adapter (programmable, e.g. Network/storage FPGA with programmable part of firmware image).
-SmartNIC enabled                                 network-smartnic        ❌                           ✅                                      Labels a host/node that includes a Programmable accelerator for vSwitch/vRouter, Network Function and/or Hardware Infrastructure.
-SmartSwitch enabled                              network-smartswitch     ❌                           ✅                                      Labels a host/node that is connected to a Programmable Switch Fabric or TOR switch
-================================================ ======================= =========================== ====================================== ================================================================================================================================================================= =============================================
++-------------------+-------------------------+---------------+---------------+------------------------+---------------+
+| Profile Extension | Mnemonic                | Applicable to | Applicable to | Description            | Notes         |
+| Name              |                         | Basic Profile | High          |                        |               |
+|                   |                         |               | Performance   |                        |               |
+|                   |                         |               | Profile       |                        |               |
++-------------------+-------------------------+---------------+---------------+------------------------+---------------+
+| Compute Intensive | compute-high-perf-cpu   | ❌            | ✅            | Nodes that have        | May use       |
```

Comment:
@CsatariGergely There are extra spaces in some lines of the table, The table is not displayed